test2.py

```diff
@@ -0,0 +1,115 @@
+import datetime
+import time
+# 1. Given a date (yyyy-mm-dd) write a function that tells you how many days it has been since that date. The function must be able to find out what the current date is.
+
+# BONUS if the date is in the future, make the function tell you how many days
+# away it is until that date.
+
+# Example:
+# input: 2022-06-22
+# output: 1 day ago (this is based on the function finding that today's date is
+# 2022-06-23 this will obviously vary based on what day it currently is)
+
+
+def timeBetweenNowAndThen(dateInput):
+    today = datetime.datetime.now()
+    todayUnix = time.mktime(today.timetuple()) * 1000
+    userYear = dateInput[:4]
+    userMonth = dateInput[5:7]
+    userDay = dateInput[8:11]
+    userYearMonthDay = datetime.date(
+        int(userYear), int(userMonth), int(userDay))
+    userUnixTime = (time.mktime(userYearMonthDay.timetuple()) * 1000)
+    unixBetweenDates = todayUnix - userUnixTime
+    unixYearsBetween = unixBetweenDates / 31_536_000_000
+    unixMonthsBetween = (unixBetweenDates -
+                         (int(unixYearsBetween) * 31_536_000_000)) / 2_629_800_000
+    unixDaysBetween = ((unixBetweenDates - (int(unixYearsBetween) * 31_536_000_000)
+                        ) - (int(unixMonthsBetween) * 2_629_800_000)) / 86_400_000
+
+    result = str(int(unixYearsBetween)), " Year(s), ", str(int(
```

Comment:
if you replace the `,` with `+` here it will make result a string and you won't need lines 33-35